tests/cypress/e2e/features/analytics_pipeline.js

```diff
@@ -194,35 +186,17 @@ context('Analytics pipeline', () => {
             cy.get('#cvat_canvas_shape_2').should('not.exist');
             cy.saveJob();
 
-            cy.logout();
-            cy.getAuthKey().then((res) => {
-                const authKey = res.body.key;
-                cy.request({
-                    method: 'POST',
-                    url: 'api/analytics/reports',
-                    headers: {
-                        Authorization: `Token ${authKey}`,
-                    },
-                    body: {
-                        project_id: projectID,
-                    },
-                }).then((response) => {
-                    const rqID = response.body.rq_id;
-                    waitForReport(authKey, rqID);
-                });
-            });
-            cy.login();
-            cy.intercept('GET', '/api/analytics/reports**').as('getReport');
-
             cy.visit(`/projects/${projectID}`);
             openAnalytics('project');
-            cy.wait('@getReport');
+            cy.get('.cvat-empty-performance-analytics-item').should('exist').and('be.visible').within(() => {
+                cy.get('button').contains('Request').click();
+            });
+
             checkCards(true);
             checkHistograms();
 
             cy.visit(`/tasks/${taskID}`);
             openAnalytics('task');
-            cy.wait('@getReport');
             checkCards(true);
```

Comment:
If we dont check empty cards with this function anymore, I suppose we can remove the `nonNull` paremeter from this function and call it just `checkCards()`